### PR TITLE
Module 9 example finished, Images to be finished

### PR DESCRIPTION
**Pull request by PINGXIANG**

### CODE REVIEW

**Comment by PINGXIANG** [comment]
The branch is amazing!
But what if I send 2 branches, for example, module 9 and module 8 individually, with each branch only changing their corresponding module before you merge. Would that causes a problem for you to merge?

File: problemsets-mat223/modules/module9.tex [hunk 1]
```diff
@@ -71,7 +71,29 @@
 	\]
 	For each of $S$ and $T$, determine whether the transformation is linear.
 
-	XXX Finish
+	For $\mathcal T$:
+	
+	Let vectors $\vec u=\mat{u_1\\u_2},\vec v=\mat{v_1\\v_2}$ and scalar $\alpha$ be given. So $\vec u +\vec v=\mat{u_1+v_1\\u_2+v_2}$ and $a\vec u = \mat{a u_1\\a u_2}$. We then have
```
Comment (siefkenj) [inline problemsets-mat223/modules/module9.tex]: `$\vec u=\mat{u_1\\u_2},\vec v=\mat{v_1\\v_2}$` -> `$\vec u=\mat{u_1\\u_2}$, $\vec v=\mat{v_1\\v_2}$`

`and` -> `, and let $\alpha$ be a scalar.`

Comment (siefkenj) [reply problemsets-mat223/modules/module9.tex]: Remove the second sentence and replace it with "We need to verify that $T(\vec u+\vec v)=T\vec u+T\vec v$ and $...$. Computing, we see"

File: problemsets-mat223/modules/module9.tex [hunk 2]
```diff
@@ -71,7 +71,29 @@
...
+	\]
+	and
+	\[
+	    \mathcal T(a\vec u)=\mat{2a u_1\\a u_2}=a\mat{2 u_1\\ u_2}=a\mathcal T(\vec u).
+	\]
+	Therefore, $\mathcal T$ is linear.
+	
+	For $\mathcal S$:
+	
+	We could easily find two counter examples such as:
```
Comment (siefkenj) [inline problemsets-mat223/modules/module9.tex]: Let's leave the cleverness out of it. Also avoid saying things are "easy". Explain this one by starting naively, i.e., computing "S(\vec u+\vec v)$ and noticing that it doesn't look like $S(\vec u)+S(\vec v)$. "Therefore, we will guess that $S$ is not linear and look for a counter example. Using $\vec u=\mat{1\\0}$ and $\vec v=\mat{0\\1}$, we see ..."

File: problemsets-mat223/modules/module9.tex [hunk 3]
```diff
@@ -71,7 +71,25 @@
 	\]
 	For each of $S$ and $T$, determine whether the transformation is linear.
 
-	XXX Finish
+	For $\mathcal S$:
+	
+	Let vectors $\vec u=\mat{u_1\\u_2}$, $\vec v=\mat{v_1\\v_2}$ and scalar $\alpha$ be given. Then we need to verify that $\mathcal S(\vec u+\vec v)=\mathcal S(\vec u)+\mathcal S(\vec v)$ and $\mathcal S(a\vec u)=a\mathcal S(\vec u)$. Computing, we see:
```
Comment (siefkenj) [inline problemsets-mat223/modules/module9.tex]: `scalar` -> `the scalar`

`Then we` -> `We`

File: problemsets-mat223/modules/module9.tex [hunk 4]
```diff
@@ -71,7 +71,25 @@
...
+	\]
+	and
+	\[
+	    \mathcal S(a\vec u)=\mat{2a u_1\\a u_2}=\mathcal S\mat{a u_1\\a u_2}=a\mat{2 u_1\\ u_2}=a\mathcal S(\vec u).
+	\]
+	Therefore, $\mathcal S$ is linear.
+	
+	For $\mathcal T$:
+	
+	Notice that $\mathcal T(\vec u+\vec v)$ doesn't look like $\mathcal S(\vec u)+\mathcal T(\vec v)$. Therefore, we will guess that $S$ is not linear and look for a counter example. Using $\xhat=\mat{1\\0}$ and $\yhat=\mat{0\\1}$, we see:
```
Comment (siefkenj) [inline problemsets-mat223/modules/module9.tex]: Expand out what `\mathcal T(\vec u+\vec v)` and `\mathcal T(\vec u)+\mathcal T(\vec v)` is. You have `\mathcal S` mentioned as a typo...

Comment (PINGXIANG) [reply problemsets-mat223/modules/module9.tex]: I have expanded out as you said. But I think that, if I expand out T(u+v) and T(u)+T(v), and show that they are not equal for all vectors, isn't that a proof for the question? Then the counter example looks duplicate for me

File: problemsets-mat223/modules/module9.tex [hunk 5]
```diff
@@ -70,8 +70,28 @@
 		\mat{x\\y}\stackrel{T}{\mapsto}\matc{x\\y+4}.
 	\]
 	For each of $S$ and $T$, determine whether the transformation is linear.
+	
+	Let vectors $\vec u=\mat{u_1\\u_2}$, $\vec v=\mat{v_1\\v_2}$ and the scalar $\alpha$ be given.
 
-	XXX Finish
+	For $\mathcal S$:
+	
+	We need to verify that $\mathcal S(\vec u+\vec v)=\mathcal S(\vec u)+\mathcal S(\vec v)$ and $\mathcal S(a\vec u)=a\mathcal S(\vec u)$. Computing, we see:
```
Comment (siefkenj) [inline problemsets-mat223/modules/module9.tex]: `a` -> `\alpha`

`we see:` -> `we see`

File: problemsets-mat223/modules/module9.tex [hunk 6]
```diff
@@ -70,8 +70,28 @@
...
+	\]
+	and
+	\[
+	    \mathcal S(a\vec u)=\mat{2a u_1\\a u_2}=\mathcal S\mat{a u_1\\a u_2}=a\mat{2 u_1\\ u_2}=a\mathcal S(\vec u).
+	\]
+	Therefore, $\mathcal S$ is linear.
+	
+	For $\mathcal T$:
+	
+	Notice that $\mathcal T(\vec u+\vec v)=\matc{u_1+v_1\\u_2+v_2+4}$ doesn't look like $\mathcal T(\vec u)+\mathcal T(\vec v)=\matc{u_1+v_1\\u_2+v_2+8}$. Therefore, we will guess that $T$ is not linear and look for a counter example. Using $\xhat=\mat{1\\0}$ and $\yhat=\mat{0\\1}$, we see:
```
Comment (siefkenj) [inline problemsets-mat223/modules/module9.tex]: `we see:` -> `we see`

Break this into a couple of lines instead of one really long one.

**Comment by siefkenj** [comment]
Great!

**Pull request merged**